### DISCUSSION TITLE
Questions about Timer.

### DISCUSSION THREAD

**Discussion by stonez56** [arendst/Tasmota#22352] [Q&A] [answered]
I have setup timer and it worked to turn the power on and off at the specific time accurately.

However, I still have three more questions about timer.
A: How does repeat work?  If I didn't tick repeat, will the timer only run once?
B: My device time zone and time is correct. How come the Sunrise and Sunset time are incorrect? How to adjust these?
C: What's the +/- 00 for? How do I use them?

![timer](https://github.com/user-attachments/assets/bd18b012-7d3e-4d0c-926b-9e29afa0a37e)

**Comment by Jason2866** [comment]
https://tasmota.github.io/docs/Timers/

**Reply by sfromis** [reply, marked as answer]
For full time zone-related settings, including coordinates for sunset/sunrise:
https://tasmotatimezone.com/

**Reply by stonez56** [reply]
Thank you for the time zone setting trick!
